templates/cli/lib/questions.js.twig

```diff
@@ -306,165 +324,344 @@ const questionsLogin = [
+            return true;
+        },
+        choices: (answers) => {
+            return answers.setup;
+        }
+    },
     {
         type: "input",
-        name: "entrypoint",
-        message: "Enter the entrypoint",
-        default: null,
-        validate(value) {
-            if (!value) {
-                return "Please enter your entrypoint";
-            }
-            return true;
+        name: "setupoverride",
+        when: (answers) => {
+            answers.setupoverride = cliConfig.force;
+
+            return false;
         }
     },
+    {
+        type: "input",
+        name: "override",
+        when: () => !cliConfig.force,
+        message: 'Would you like to override existing topics? This can lead to loss of data! Type "YES" to confirm.'
+    }
 ]
 
 const questionsPushTeams = [
     {
-        type: "checkbox",
-        name: "teams",
-        message: "Which teams would you like to push?",
-        validate: (value) => validateRequired('team', value),
-        choices: () => {
+        type: 'checkbox',
+        name: 'setup',
+        async when(answers) {
             let teams = localConfig.getTeams();
             if (teams.length === 0) {
                 throw new Error("No teams found in the current directory. Run `appwrite pull team` to fetch all your teams.");
             }
-            return teams.map(team => {
+
+            answers.setup = teams.map(team => {
                 return {
                     name: `${team.name} (${team['$id']})`,
                     value: team.$id
                 }
             });
+
+            if (Array.isArray(cliConfig.ids) && cliConfig.ids.length !== 0) {
+                answers.setup = answers.setup.filter((team) => cliConfig.ids.includes(team.value));
+            }
+
+
+            return false;
+        }
+    },
+    {
+        type: "checkbox",
+        name: "teams",
+        message: "Which teams would you like to push?",
+        validate: (value) => validateRequired('team', value),
+        when: (answers) => {
+            if (cliConfig.all || Array.isArray(cliConfig.ids) && cliConfig.ids.length !== 0) {
+                answers.teams = answers.setup;
+                return false;
+            }
+
+            return true;
+        },
+        choices: (answers) => {
+            return answers.setup;
+        }
+    },
+    {
+        type: "input",
+        name: "setupoverride",
+        when: (answers) => {
+            answers.setupoverride = cliConfig.force;
+
+            return false;
         }
     },
     {
         type: "input",
         name: "override",
+        when: () => !cliConfig.force,
         message: 'Are you sure you want to override this team? This can lead to loss of data! Type "YES" to confirm.'
-    },
+    }
 ];
 
+const questionsGetEntrypoint = [
+    {
+        type: "input",
+        name: "entrypoint",
+        message: "Enter the entrypoint",
+        default: null,
+        validate(value) {
```

Comment:
Is there a way to add an example param here like a hint? 

Else maybe we can do `message: "Enter the entrypoint (example: src/index.js )",`

Reply:
For that we will need to fetch the runtime from the appwrite instance.